src/libraries/System.Private.CoreLib/src/System/Buffers/Binary/ReaderBigEndian.cs

```diff
@@ -96,10 +118,27 @@ public static ulong ReadUInt64BigEndian(ReadOnlySpan<byte> source)
             return result;
         }
 
+        /// <summary>
+        /// Reads a Double out of a read-only span of bytes as big endian.
+        /// </summary>
+        /// <returns>If the span is too small to contain a Double, return false.</returns>
```

Comment:
Same doc comments here about consistency and also add the other tags. Use `langword`.

```suggestion
        /// <returns><see langword="true" /> if the span is large enough to contain an <see cref="System.Double" />; otherwise, <see langword="false" />.</returns>
```